hd/address.go

```diff
@@ -282,9 +282,20 @@ func CalcSha512(buf []byte) []byte {
 }
 
 func ReverseBytes(buf []byte) []byte {
-	res := []byte{}
-	for i := len(buf) - 1; i >= 0; i-- {
-		res = append(res, buf[i])
+	var res []byte
```

Comment:
Maybe this belongs in tmlibs/common/bytes.go?

Reply:
Yeah, a nice reverse function... sounds good